cli/graphql.md

```diff
@@ -2358,6 +2439,118 @@ mutation Delete($noteId: ID!) {
 }
 ```
 
+## Automatically Import Existing DataSources
+
+The Amplify CLI currently supports importing serverless Amazon Aurora MySQL 5.6 databases. Follow these instructions to get started:
```

Comment:
Is this the min version we support or is this the only version that we support for now?

Reply:
AppSync currently only supports Aurora serverless MySQL 5.6 clusters with the data api enabled.